doc/build/sysbuild/index.rst

```diff
@@ -222,6 +223,34 @@ build system to the value ``BAR``, run the following commands:
 
          cmake -Dmy_sample_FOO=BAR ...
 
+Naturally, if you are configuring additional applications in sysbuild, named
+e.g. ``sample_a``, ``sample_b``, and ``sample_c``, then you can set a different
+value of ``FOO`` in each of them using::
+
+  -Dsample_a_FOO=FOO -Dsample_b_FOO=BAR -Dsample_c_FOO=BAZ
+
+On the other hand, suppose you want to set ``FOO`` to the same value in all of
+these samples. Then, instead of repeating yourself, you can simply pass::
+
+  -Dall_FOO=BAR
+
+The ``all`` namespace is shared by all domains in a given multi-image build.
+It can be used to simplify global configuration of images.
+
+.. note::
+
+   CMake variables processed by sysbuild are not automatically accessible in
+   Zephyr build systems. Rather than initializing each domain's CMake cache,
+   sysbuild generates a separate cache file, which is then loaded inside
+   ``find_package(Zephyr)``. Therefore, when using sysbuild, common variables
+   like ``BOARD`` will not be populated before that point.
```

Comment:
I like the suggestion, but do you think we should at least introduce the "sysbuild cache" concept here as well? It comes up a fair amount in conversation.

Reply:
> but do you think we should at least introduce the "sysbuild cache" concept here as well? It comes up a fair amount in conversation.

you have a fair point there. I've actually noticed the same when talking to people.
Maybe it would be best to have an independent subsection somewhere with a brief description of the sysbuild cache.
That way we keep the `note` short and precise, which is always a good idea, while at the same time have a dedicated subsection for the sysbuild cache mechanism which makes it easier to know where to update if we change the sysbuild cache mechanism itself.

If you like that proposal then such a sysbuild cache description can be part of this PR or independent, as you find fit.

Reply:
you can even copy a direct link to such subsection into next conversation, something not possible with a `note` 😉